Resources/views/Form/formatter.html.twig

```diff
@@ -49,6 +49,14 @@
                         appendClass = 'html';
                         break;
                     case 'richhtml':
+                        {%- for name, plugin in ckeditor_plugins if ckeditor_plugins is defined -%}
```

Comment:
@rande @roukmoute tells me there is no global configuration change with this P.R. The code you see here is supposed to act on the current instance only. What makes you think otherwise ?